lib/ansible/cli/__init__.py

```diff
@@ -169,32 +175,33 @@ def run(self):
 
         # warn about deprecated config options
         for deprecated in C.config.DEPRECATED:
-            name = deprecated[0]
-            why = deprecated[1]['why']
+            name: str = deprecated[0]
+            why: str = deprecated[1]['why']
+            alt: str
             if 'alternatives' in deprecated[1]:
                 alt = ', use %s instead' % deprecated[1]['alternatives']
             else:
                 alt = ''
-            ver = deprecated[1].get('version')
-            date = deprecated[1].get('date')
-            collection_name = deprecated[1].get('collection_name')
+            ver: str | None = deprecated[1].get('version')
+            date: str | None = deprecated[1].get('date')
+            collection_name: str | None = deprecated[1].get('collection_name')
             display.deprecated("%s option, %s%s" % (name, why, alt),
                                version=ver, date=date, collection_name=collection_name)
 
     @staticmethod
-    def split_vault_id(vault_id):
+    def split_vault_id(vault_id) -> Tuple[str | None, ...]:
```

Comment:
```suggestion
    def split_vault_id(vault_id: str) -> tuple[str | None, ...]:
```

It looks like `vault_id` should be a string?

Reply:
Yes! 👍